libfwupd/fwupd-remote.c

```diff
@@ -1561,8 +1561,10 @@ fwupd_remote_load_signature(FwupdRemote *self, const gchar *filename, GError **e
 
 	/* load JCat file */
 	gfile = g_file_new_for_path(filename);
-	if (!jcat_file_import_file(jcat_file, gfile, JCAT_IMPORT_FLAG_NONE, NULL, error))
+	if (!jcat_file_import_file(jcat_file, gfile, JCAT_IMPORT_FLAG_NONE, NULL, error)) {
+		fwupd_error_convert(error);
```

Comment:
Is this the right level? I would think it should be just one time; right before stuff goes over the wire.

Reply:
Ok